Sources/KSCrashRecordingCore/KSCxaThrowSwapper.c

```diff
@@ -256,7 +257,11 @@ static void perform_rebinding_with_section(const section_t *section,
     if (isDataConst)
     {
         oldProtection = get_protection(indirect_symbol_bindings);
-        mprotect(indirect_symbol_bindings, section->size, PROT_READ | PROT_WRITE);
+        if (mprotect(indirect_symbol_bindings, section->size, PROT_READ | PROT_WRITE) != 0)
+        {
+            perror("mprotect failed");
```

Comment:
Maybe use `KSLOG_TRACE` instead? The default log level is error but this would spam in console if reported as error (and I don't think we need this reported to KSCrash users, right?)